legacy-search.php

```diff
@@ -7,26 +7,50 @@
 <head>
   <?php include_once('common/header.php'); ?>
   <title>Search</title>
+  <style>
+    #main.disabled:before {
+      content: "";
+      position: fixed;
+      z-index: 1;
+      min-height: 100vh;
+      width: 100%;
+      left: 0;
+      top: 0;
+      background: rgba(0,0,0, .5);  
+    }
+    #announcement {
+      position: fixed;
+      z-index: 2;
+      border: 2px #ccc solid;
+      padding: 30px;
+      left: 50%;
+      top: 50%;
+      transform: translate(-50%, -50%);
+    }
+  </style>
 </head>
 
 <body id="mainSearch">
   <?php include_once('common/nav.php'); ?>
-  <div id="main" class="main grid-container">
+  <div id="announcement" style="background-color:#6e2424;opacity:0.9;color:white;padding:10px;">
+    <strong>The London Stage Database team has retired the Legacy Search on May 1, 2025.</strong> Please take a moment before that date to reproduce any pre-2021 searches and export any resulting datasets you may wish to preserve for future use. We are making this change in order to free up computational resources for new features and data, currently in development with support from the National Endowment for the Humanities. Watch this space for more updates and, coming soon, new ways to keep up with the latest project developments!
```

Comment:
@mattieburkert Do you have some copy to describe the retirement of Legacy Search? The only way users will be able to see this page after the retirement is by going to the old /legacy-search.php URL (from a bookmark presumably), but the current phrasing incorrectly implies casual users will still be able to conduct legacy searches.

Reply:
@uo-zz Mattie has proposed the following announcement text:
`The Legacy Search feature has been retired. To learn more, please visit our <a>blog at https://blogs.uoregon.edu/londonstage/2025/05/07/legacy/</a>.`